src/VaultClient.ts

```diff
@@ -0,0 +1,276 @@
+// @ts-ignore
+import { masterVault as backend } from '@xbacked-dao/xbacked-contracts';
+import { Vault } from './Vault';
+import { convertToMicroUnits, calculateInterestAccrued } from './utils';
+import { Account } from './Account';
+import { AbiInterface, AccountInterface, UserVaultReturnParams, VaultReturnParams } from './interfaces';
+
+export class VaultClient extends Account {
+ 
```

Comment:
@AwolDes 
Same thing I said about the naming convention for the StakingClient applies to the VaultClient